src/lib/load.js

```diff
@@ -2,6 +2,7 @@ const { Client } = require("./es");
 const get_logger = require("./logger");
 const dot = require("dot-object");
 const Mustache = require("mustache");
+const { option, options } = require("yargs");
```

Comment:
Are these being used? I just noticed them here but we also have "options" as the 2nd passed arg into "load' below so I don't think `yargs.options` is reachable, at least?

Reply:
Not sure how this got added